### PR TITLE
Added Dutch translation to meta-video

### PR DESCRIPTION
**Pull request by OpenELEQ**

### CODE REVIEW

File: plugin.video.meta/resources/language/Dutch/strings.po [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+msgid "Automatic movies library server"
+msgstr "Automatische film bibliotheek server"
+
+msgctxt "#32305"
+msgid "Set added time to release date"
+msgstr "Zet tijd van toevoegen bij uitgave datum"
+
+msgctxt "#32402"
+msgid "Use simple selection dialog"
+msgstr "Gebruik simpele selectie dialoog"
```
Comment (ghost) [inline plugin.video.meta/resources/language/Dutch/strings.po]: wouldn't "Gebruik simpel selectie dialoog" be better?

File: plugin.video.meta/resources/language/Dutch/strings.po [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+msgid "Hide information dialogs"
+msgstr "Verberg informatie dialogen"
+
+msgctxt "#32413"
+msgid "Hide keyboard dialog"
+msgstr "Verberg keyboard dialogen"
+
+msgctxt "#31000"
+msgid "Would you like to automatically set Meta as a movies video source?"
+msgstr "Wilt u Meta automatisch als film bron instellen"
```
Comment (ghost) [inline plugin.video.meta/resources/language/Dutch/strings.po]: missing a ?

File: plugin.video.meta/resources/language/Dutch/strings.po [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+msgid "Do you want to remove your existing players first?"
+msgstr "Wilt u uw huidige spelers eerst verwijderen?"
+
+msgctxt "#31007"
+msgid "Enter password"
+msgstr "Vul wachtwoord in"
+
+msgctxt "#31008"
+msgid "Players updated"
+msgstr "Spelers geüpdate"
```
Comment (ghost) [inline plugin.video.meta/resources/language/Dutch/strings.po]: woudn't "Spelers bijgewerkt" be better?

File: plugin.video.meta/resources/language/Dutch/strings.po [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+msgid "Blockbusters"
+msgstr "Kaskrakers"
+
+msgctxt "#31014"
+msgid "On the air"
+msgstr "Nu op TV"
+
+msgctxt "#31015"
+msgid "Update players"
+msgstr "Spelers updaten"
```
Comment (ghost) [inline plugin.video.meta/resources/language/Dutch/strings.po]: wouldn't "Spelers bijwerken" be better?

File: plugin.video.meta/resources/language/Dutch/strings.po [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+msgid "Popular"
+msgstr "Populair"
+
+msgctxt "#31022"
+msgid "Select default player"
+msgstr "Selecteer standaard speler"
+
+msgctxt "#31023"
+msgid "Failed to update players"
+msgstr "Updaten spelers mislukt"
```
Comment (ghost) [inline plugin.video.meta/resources/language/Dutch/strings.po]: woudn't "Bijwerken spelers mislukt" be better?

**Comment by metate** [comment]
Thanks!

**Pull request merged**